### ISSUE TITLE
template-lint "no-partial" trips up when partial is passed as param

### ISSUE THREAD

**Issue by abhishek1nair** [ember-template-lint/ember-template-lint#2893]
# How to reproduce?
```
<Component
  @param={{"partial"}}
 ....
/>
``` 
This syntax is triggering the "no-partial" template error.
